src/Integrations/Integrations/Curl/CurlIntegration.php

```diff
@@ -69,8 +70,13 @@ public function init()
                 $info = \curl_getinfo($ch);
                 $sanitizedUrl = \DDTrace\Util\Normalizer::urlSanitize($info['url']);
                 $normalizedPath = \DDTrace\Util\Normalizer::uriNormalizeOutgoingPath($info['url']);
+                $host = Urls::hostname($sanitizedUrl);
                 unset($info['url']);
 
+                if ($host) {
```

Comment:
Checking `Urls::hostname` it will never fail, so this if is meaningless, unless you mean to avoid parse failures?
However, this needs to check for `$host != "unknown-host" && $host != "unparsable-host"`.

Reply:
Good catch. I think there is still value in reporting which services (in the service map) reports to the `unknown/unparseable-host`, so I think I'll add it in any case, rather than pass silently.